### PR TITLE
[14.0] [mig] web_widget_numeric_step

### PR DESCRIPTION
**Pull request by fshah-initos**

### CODE REVIEW

**Comment by legalsylvain** [comment]
CC : @quentinDupont

**Comment by fshah-initos** [comment]
It will be nice if we get the reviews here

**Comment by fshah-initos** [comment]
It will be nice if we get the reviews here

**Comment by fshah-initos** [comment]
It will be nice if we get the reviews here

**Comment by fshah-initos** [comment]
It will be nice if we get the reviews here

**Comment by fshah-initos** [comment]
It will be nice if we get the reviews here

**Comment by hkapatel-initos** [comment]
It will be nice if someone gets the reviews here.

**Comment by hkapatel-initos** [comment]
@chienandalu and @sergio-teruel It would be great if you both can review it.

File: web_widget_numeric_step/README.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/web <https://github.com/OCA/web/tree/13.0/web_widget_numeric_step>`_ project on GitHub.
+
+You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
```
Comment (sergio-teruel) [inline web_widget_numeric_step/README.rst]: Please. You must re-genarate this file

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by hkapatel-initos** [comment]
It will be nice if someone gets the reviews here.

**Comment by chienandalu** [comment]
@hkapatel-initos You have to attend comments when you receive them and @sergio-teruel's one is not although you thumbed it up.

Also: please, don't randomly ping for reviews. The usual procedure would be to ping the original contributors first (neither me or @sergio-teruel are on that list) and give some reviews in exchange for yours.

File: web_widget_numeric_step/README.rst (hunk 1 above)
Comment (hkapatel-initos) [reply web_widget_numeric_step/README.rst]: hello, @sergio-teruel can you please re-check?

**Comment by chienandalu** [comment]
@Tardo Can you take a look?

File: web_widget_numeric_step/README.rst (hunk 1 above)
Comment (Tardo) [reply web_widget_numeric_step/README.rst]: Regenerate to show your name in "Contributors" section ;)

**Review by Tardo — APPROVED** [review]
LGTM +1

**Comment by hkapatel-initos** [comment]
https://github.com/OCA/web/pull/1977#discussion_r705300471
Hello @Tardo Done.

**Review by hkapatel-initos — APPROVED** [review]

**Comment by hkapatel-initos** [comment]
Hello maintainers,
it would be nice if you try to merge it.

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**